### ISSUE TITLE
Unable to retrieve scripts with Proxy Route Prefix setup

### ISSUE THREAD

**Issue by jeebs95** [ElfFriend-DnD/foundryvtt-confetti#6]
When foundry set to have a routePrefix the confetti module isn't able to grab the /scripts/greensock/esm/all.js

**Comment by jeebs95** [comment]
So I have found a fix that seems to work. It is based on the setup FX Master since they also use greensock.

For module.json
```
"esmodules": [
		"greensock/esm/all.js",
		"foundryvtt-confetti.js"
	]
```

And in Confetti.js altering the from line and it works with the proxy routePrefix!

`from '../../../../scripts/greensock/esm/all.js';`

**Comment by akrigline** [comment]
Oof, that's unfortunate. Thanks for investigating! If you open a pr I'll merge it and get a release out this weekend.

**Comment by jeebs95** [comment]
PR made. Yeah my players were sad they couldn't do Confetti last night. First game after getting a proxy setup and it is one of our favorite modules. It is a staple for all our games.

**Comment by akrigline** [comment]
Merged and released with 1.0.3
